modules/caddyhttp/tracing/README.md

```diff
@@ -0,0 +1,43 @@
+# Tracing module
```

Comment:
Not against a readme here, but I think this is the only folder in this repo that has one (other than the root). Should we move this to the site instead so people will actually see it?

Reply:
@mholt, I agree with you. I remove it from here and will create a PR to the website repo.